### DISCUSSION TITLE
No loader is configured for ".scss"

### DISCUSSION THREAD

**Discussion by billypap1** [remix-run/remix#8868] [Q&A]
I try to migrate my Vite App to Remix but I get this error: 
**_No loader is configured for ".scss" files:_**

How can I add a scss loader to Remix?

**Comment by sergiodxa** [comment]
If you are using the Vite plugin for Remix, you can add another Vite plugin for SCSS
If you’re not using the Vite plugin there’s no way to customize the compiler and you will have to run SCSS in a separate process to output the CSS that your Remix app can import.

**Reply by billypap1** [reply]
Thank you @sergiodxa, I used Vite plugin and I don't get any issues for the .scss, .less files.

I follow this guide to migrate my app to Remix -> https://remix.run/docs/en/main/guides/migrating-react-router-app#ensure-your-app-uses-react-router-v6 but I get these errors:
 
1) **You made a GET request to \"/\" but did not provide a `loader` for route \"routes/_index\", so there is no way to handle the request.**

2) **You made a GET request to \"/login\" but did not provide a `loader` for route \"routes/$\", so there is no way to handle the request.**

The files app/entry.server.jsx, app/entry.client.jsx and app/root.jsx are the same like the guide. 

-> app/routes/_index.jsx
`export { default } from "../old-app/index";`

-> app/routes/$.jsx
`export { default } from "../old-app/index";`

and old-app/index: 

```
function AppWithCallbackAfterRender() {
  return   (<Router  />)
}
```
*React-Router v6.

It's like it doesn't pass the route to old-app. Any idea? 

UPDATE: After some modifications, I don't get the errors but I get a blank screen. In the network tab it doesn't load any js code.
